string_manipulation.rb

```diff
@@ -52,16 +59,16 @@ def string_reverse(my_string)
 
```

Comment:
The method above creates a new string that has the characters in reverse order of the original string parameter passed. This method does not change the original string. Since the specifications and the test below check for in place string reversal, this method won't pass the tests.